doc-Provisioning_Virtual_Machines_and_Hosts/topics/Provisioning_Requests.adoc

```diff
@@ -216,23 +234,42 @@ image:2323.png[]
 
 Add a customization template to provide *Kickstart*, *Cloud-Init*, or *Sysprep* files for the initial loading of the operating system.
 
+[NOTE]
+====
+ISO provisioning is performed using a Kicksart file. The customization template is required to provide Kickstart files for initial uploading of the operating system.
+====
+
+==== Cloud-Init
+
 .Cloud-Init Requirements
 * When creating a template using Red Hat Virtualization, install the *cloud-init* package on the source virtual machine. This enables Cloud-Init to source configuration scripts when a virtual machine built on that template boots.
 * See https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Virtualization/3.6/html-single/Virtual_Machine_Management_Guide/index.html#sect-Using_Cloud-Init_to_Automate_the_Configuration_of_Virtual_Machines[Using Cloud-Init to Automate the Configuration of Virtual Machines] in the Red Hat Virtualization _Administration Guide_ for more information on using Cloud-Init in a Red Hat Virtualization environment.
 * See the http://cloudinit.readthedocs.org/en/latest/[Cloud-Init Documentation] web site for example scripts.
 
+==== Kickstart
+ 
 .Kickstart Requirements for ISO Provisioning
-* The *Kickstart* file must be named *ks.cfg*.
-* Set the new virtual machine to power down after provisioning is complete.
-* {product-title_short} must use the virtual machine payload feature of Red Hat Virtualization to create a floppy disk containing the data from the selected customization template.
-* Customize the installer to include the data written to the floppy disk payload.
+* The *Kickstart* file in the root directory of the ISO must be named *ks.cfg*.
+* ISO Provisioning on Red Hat Virtualization will write the content generated by the customization template to the file named *ks.cfg* that will be attached to the virtual machine using the RHV floppy payload. If this content is to be included in the installation, the ISO image must be aware of this and include the contents of the file. This requires remastering of the ISO to include a custom Kickstart file. 
```

Comment:
> If this content is to be included in the installation, the ISO image must be aware of this and include the contents of the file.
This sentence actually contradicts the previous one. Just read it very carefully. The first one says that **RHV will write the content...to the file name ks.cfg**. The second one says that **the content must already be present**. Now as far as I understand it, the second sentence is just false. The ISO image **does not need to** include the customization template itself , it just needs to include ks.cfg which, in turn, will import content of the customization template (by using kickstart directive `%include`)

Reply:
Thanks for the clarification here @jan-zmeskal. I've revised the text to entirely remove the second sentence and instruction to remaster the ISO has been moved to the first bullet pointed requirement.